config/etfuturum/modcompat.cfg

```diff
@@ -0,0 +1,46 @@
+# Configuration file
+
+##########################################################################################################
+# misc
+#--------------------------------------------------------------------------------------------------------#
+# When modded deepslate or raw ore support is enabled, OreDictionary tags will be given to the needed Lord of the Rings ores since they don't have any tags for some reason.
+# LoTR Ores that have no raw/deepslate modded alternative will not be tagged.
+##########################################################################################################
+
+misc {
+    # Disables just the andesite, granite, and diorite full blocks, but leaves their stairs and slabs, etc enabled.
+    # Useful if you have mods like Chisel or Botania which feature these same stones but not the stairs and other variants. [default: false]
+    B:disableBaseBountifulStonesOnly=false
+
+    # Disables copper ingots and ores, but leaves the blocks and other stuff.
+    # Useful if you prefer another mod's copper, but want to use the oxidizing copper building blocks. [default: false]
+    B:disableCopperOreAndIngotOnly=false
+
+    # Adds compat for Baubles Expanded. Does nothing for standard baubles, this REQUIRES Baubles Expanded! It uses the new "wings" slot added by the expanded version. https://modrinth.com/mod/baubles-expanded
+    # When enabled, this allows the player to equip an elytra with a chestplate, by placing the elytra in a "wings" slot instead of the chestplate slot. Note that the player can only equip one elytra at a time.
+    # 0 = No compat, do not allow the elytra to be placed in a wings slot.
+    # 1 = Elytra will be placeable in a wings slot. Will enable the slot, if it isn't there.
+    # 2 = The elytra can ONLY go in the wings slots, not the chestplate slot. [range: 0 ~ 2, default: 1]
+    I:elytraBaublesExpandedCompat=1
+
+    # Deepslate ores for modded ores. Adds a set of "general" deepslate ores for common metals like "oreAluminium", "oreTin", etc, as well as explicit support for numerous mods. [default: true]
+    B:moddedDeepslateOres=true
```

Comment:
thats unlikely to truely work with gt. should probably be done in gt instead

Reply:
AFAICT it wasn't spawning deepslate ores anyway. I'll set this to false, though. As for enabling deepslate ore in GTNH, I think it will require some reworking of the ore tile entity. I'd assume we want to do this for nether ores in basalt + blackstone but we don't have enough bits to add 3 new stone types to the ore tile entity.

Reply:
yea adding this in gt is not mandatory at all. I am just saying *if* we do it then it should be there so it applies to all the ores.